docs/language_specification/tokens/literals.md

```diff
@@ -0,0 +1,28 @@
+| Literal | Size / precision          |
+|:--------|:--------------------------|
+| Integer | 64-bit signed             |
+| Float   | IEEE 754 double precision |
```

Comment:
Should we add character literals to this table? Maybe that doesn't make sense...

Reply:
Maybe. This entry has to be updated. I'll have a look.

Reply:
We don't really have character literals as tokens, my fault.

Letters (alphabetical characters plus underscore) are only helpers for constructing an identifier.

Other character literals turn directly into other types of tokens, such as operators and punctuators, newlines, and so on.